config/prow/config_knative.yaml

```diff
@@ -151,6 +151,18 @@ presubmits:
         - "./test/presubmit-integration-tests-latest-release.sh"
       dot-dev: true
 
+  knative/client-contrib:
+    - repo-settings:
+      go112-branches: []
```

Comment:
For now we'll have to add a dummy branch here to make the config-generator work :( 
See https://github.com/knative/test-infra/blob/810ad47815885a5f2ceee52539d4aef4ec4486d5/config/prow/config_knative.yaml#L287